Makefile

```
@@ -414,7 +414,7 @@ govet: ## Run govet on Go source files in the repository.
 	$(QUIET) $(GO_VET) ./...
 
 golangci-lint: ## Run golangci-lint
-ifneq (,$(findstring $(GOLANGCILINT_WANT_VERSION),$(GOLANGCILINT_VERSION)))
+ifneq (,$(findstring $(GOLANGCILINT_VERSION),$(GOLANGCILINT_WANT_VERSION)))
```

Another option would be to use `make`'s [substitution reference](https://www.gnu.org/software/make/manual/html_node/Substitution-Refs.html) to strip of the `v` from `$(GOLANGCILINT_WANT_VERSION)`:

```suggestion
ifneq (,$(findstring $(GOLANGCILINT_WANT_VERSION:v%=%),$(GOLANGCILINT_VERSION)))
```

(this is what we [use in the cilium-cli repo](https://github.com/cilium/cilium-cli/blob/a681b5e588e9542685070f00749fdaefc2737c57/Makefile#L86))

---

thanks - good idea. Especially in combination with `golangci-lint version --format short` that would even allow for an equal comparison (instead of the `findstring`)) :thinking:

---

I switched to your proposed option - but kept the short format.

---

Yeah, the short format is indeed safer (e.g. if the Go version included in the long format would match at some point :smile:). Will change to the short format in cilium-cli as well.

---

great. thanks Tobias!